relay/config.example.yml

```
@@ -8,6 +8,6 @@ processing:
   enabled: true
   kafka_config:
     - {name: "bootstrap.servers", value: "kafka:9092"}
-    - {name: "message.max.bytes", value: 50000000} #50MB or bust
+    - {name: "message.max.bytes", value: 50000000}
```

Alternatively, we could also leave the "50MB" there.

```suggestion
    - {name: "message.max.bytes", value: 50000000} # 50MB
```